spec/rubocop/cop/rails/rake_environment_spec.rb

```diff
@@ -83,4 +83,10 @@
       task default: :spec
     RUBY
   end
+
+  it 'does not register an offense to task with no block' do
+    expect_no_offenses(<<~RUBY)
+      task(:spec).clear
```

Comment:
This is a nitpick. `do_something` will be a more generic naming than `clear`.

Reply:
Thanks! I updated to `task(:spec).clear` to `task(:foo).do_something`